src/runtime_src/core/pcie/driver/linux/xocl/xocl_subdev.c

```diff
@@ -986,6 +1029,25 @@ void xocl_subdev_destroy_all(xdev_handle_t xdev_hdl)
 	xocl_unlock_xdev(xdev_hdl);
 }
 
+void xocl_subdev_destroy_by_level_skip_cus(xdev_handle_t xdev_hdl, int level)
```

Comment:
*_skp_cus is not a common interface. you may create something like xocl_subdev_destroy_by_level(..., int skip_devices[])

Reply:
Implemented differently. While creating a subdevice, we have added a slot id information for each subdevice. While destroying the devices then used that slot information